packages/fluentui/react-northstar/src/components/Tree/TreeTitle.tsx

```diff
@@ -85,11 +116,18 @@ const TreeTitle: React.FC<WithAsProp<TreeTitleProps>> & FluentComponentStaticPro
       level,
       index,
       treeSize,
+      selected,
+      selectable,
```

Comment:
can you get `selectableParent` here as well?

Reply:
added :)